pkgs/development/python-modules/pydyf/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib,
+  buildPythonPackage,
+  fetchPypi,
+  isPy3k,
+  coverage,
+  ghostscript,
+  pillow,
+  pytest,
+  pytest-cov,
+  pytest-flake8,
+  pytest-isort
+}:
+
+buildPythonPackage rec {
+  pname = "pydyf";
+  version = "0.1.2";
+  disabled = !isPy3k;
+
+  # setup.py is auto-generated and doesn't pass the flake8 check
+  checkPhase = ''
+    runHook preCheck
+    pytest --ignore=setup.py
```

Comment:
This could probably use `pytestCheckHook`: https://nixos.org/manual/nixpkgs/stable/#using-pytestcheckhook

Reply:
Thanks, updated in both `pydyf` and `weasyprint`!